docs/resources/Application.md

```diff
@@ -48,7 +49,7 @@
   },
   "primary_sku_id": "172150183260323840",
   "slug": "test",
-  "summary": "This is a game",
+  "summary": "", // field will be removed in v11
```

Comment:
This is an invalid JSON (JSON documents cannot have comments). I suggest removing this line from the example completely.
```suggestion
```